.gitignore

```diff
@@ -6,3 +6,7 @@ public/bundle.js
 config/config.js
 # jest coverage
 coverage
+#MockData used for ProductOverview component
+MockData
```

Comment:
Do we still need this if the file was deleted?

Reply:
It's not deleted on my local repo, I am thinking I might still need mockdata for future tests.

Reply:
It's not hurting anything, so we can leave it.